packages/utilities-react/src/StepFlow/StepContext.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useContext, createContext, ReactNode, useState } from 'react';
+import { StepContextType } from './types';
+
+const StepContext = createContext<StepContextType | undefined>(undefined);
+
+interface StepProviderProps {
+  children?: ReactNode;
```

Comment:
Should this prop be required? In `StepProvider`, when you destructure it, there’s no handling for the case where it’s `undefined`. Also, would the provider actually do anything if it were `undefined`?